examples/run_setup_tutorial.py

```diff
@@ -0,0 +1,100 @@
+"""
+=========
+Run Setup
+=========
+
+By: Jan N. van Rijn
+
+One of the key features of the openml-python library is that is allows to
+reinstantiate flows with hyperparameter settings that were uploaded before.
+This tutorial uses the concept of setups. Although setups are not extensively
+described in the OpenML documentation (because most users will not directly
+use them)
+
+A key requirement for reinstantiating a flow is to have the same scikit-learn
+version as the flow that was uploaded. This tutorial will upload the flow
```

Comment:
Don't these two sentences contradict each other?

Reply:
Sorry, I also don't get your updated version. You're saying it is important to have the same scikit-learn version, but then say this doesn't matter because the tutorial uploads the flow. I simply don't get how this makes the version not matter.

Reply:
I tried to reformulate. Please let me know if this is any better.